docker-compose-e2e-readme.md

```diff
@@ -0,0 +1,98 @@
+# Opzetten/Opstarten containerized BRP Bevragen end-to-end test omgeving
+
+Start de containers met behulp van de volgende statement
+
+```
+docker-compose -f docker-compose-e2e.yml up -d
+```
+
+Stop de containers met behulp van de volgende statement
+
+```
+docker-compose -f docker-compose-e2e.yml down
+```
+
+## Beheren van de BRP bevragen test database
+
+Dit kan worden gedaan met behulp van de pgAdmin tool. Deze wordt opgestart door in een browser naar de volgende url te navigeren
+
+```
+http://localhost:5433
```

Comment:
Moet je pgadmin / Postgres’s eerst lokaal hebben geïnstalleerd? Of roep je hier pgadmin aan op de docker die je net hebt gestart?